qa/tasks/cephfs/test_exports.py

```diff
@@ -114,6 +116,8 @@ def test_export_pin(self):
             self._wait_subtrees(status, 0, [('/1', 0), ('/1/4/5', 1), ('/a', 1), ('/a/b/aa/bb', 0)])
 
         # Test getfattr
+	if (str(self.mount_a.client_remote.run(args=['uname', '-r'])) < "5" and isinstance(self.mount_a, KernelMount) and not(self.mount_a.getfattr("1", "ceph.dir.pin"))):
```

Comment:
This conditional is too long. Please pull the `uname -r` out a line above and only do the test if the mount is a `KernelMount`. IOW, the `uname -r` is not necessary if we're testing ceph-fuse. Something like:

````
if kernel mount
  uname -r
  if test getfattr fails and version < v5.1
    raise skip test
````